stdlib/ballerina-builtin/src/main/ballerina/ballerina/io/natives.bal

```diff
@@ -87,4 +87,4 @@ public native function <ByteChannel channel> close();
 @Description {value:"Function to check whether next record is available or not"}
 @Param {value:"channel: The TextRecordChannel to read text records from"}
```

Comment:
Shall we replace here as well?

Reply:
c65b5580d110f274fb2a630c136996a31c17d89e